FileIO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -71,21 +72,25 @@ MeshLib::NodePartitionedMesh* NodePartitionedMeshReader::read(
 
     MeshLib::NodePartitionedMesh* mesh = nullptr;
 
+    /// Prefix of the filename of the partitioned mesh files.
+    std::string const filename_prefix =
+        boost::erase_last_copy(filename_prefix, ".vtk");
```

Comment:
The object `filename_prefix` is created here and at the same time used as an argument to the method. Is this working?